.eslintrc.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     'no-useless-escape': 'off',
     'object-curly-newline': 'off',
     'lines-between-class-members': 'off',
-    'unused-imports/no-unused-imports-ts': 'error',
+    'unused-imports/no-unused-imports-ts': 'warn', // should be warned and allowed for compilation
```

Comment:
Why do you think it should be?

Reply:
When you do the heavy refactoring and removing the pieces of code some functions will become redundant. These imports need to be fixed when creating a PR for sure. However, we should not display an error when the code is perfectly compilable. It breaks focus of the developer, feedback-loop from the IDE and browser downgrading the developer's performance when coding and making changes. If we mark these as warnings these issues could be fixed as a last step.

Reply:
Thanks, I like it